### ISSUE TITLE
Support multiple commanders

### ISSUE THREAD

**Issue by neotron** [msarilar/EDEngineer#7]
The LoadGame event, and perhaps others, include the commander name. I have multiple commanders which I'd obviously like to track separately (otherwise it'd get out of sync a lot and in general not be useful cross-account).

Since you can run multiple Elite at the same time, it would have to keep track of multiple files and log accordingly to fully support this.

**Comment by neotron** [comment]
So this is how it works:

Each journal file gets one or more LoadGame events - this event contains the commander name. If you start a second game, in parallel or not, another file is created. So you don't appear to ever mix commanders in the same file but you might have multiple load events (if you relog). Example event:

`{ "timestamp":"2016-11-05T17:33:40Z", "event":"LoadGame", "Commander":"CMDR NAME", "Ship":"Python", "ShipID":0, "GameMode":"Group", "Group":"...", "Credits":55142960, "Loan":0 }
`

**Comment by dorimanx** [comment]
Dont want to open new issue, just a question:
How can we get updated values from one PC to other. in both PC i have the game with same account.
spent some time to correct all data/mats/cargo on one PC, now i need to do it again on other.
Any way to copy some db file and be happy?
Thanks.

**Comment by msarilar** [comment]
@neotron I got something that's working; wondering what to do regarding filters and favorites/ignored blueprints. do we want to share these across commanders or should it be specific for each one?

I'm thinking it should be independent and that's how I did it, but we never know.

**Comment by neotron** [comment]
That's a good question. I'm thinking there's pros and cons for both options. For example I tend to have favorite mods (and filtered ones that I'll never use) that would be nice to share. However it might be nice in some cases to have separate goals. 

Independent, if you have it already, works fine. It's not exactly terribly difficult to favorite/filter a few things and the benefits of being able to separate it when desired is nice. My alt account is much MUCH further behind (have one engineer unlocked, ranked to 3 or 4) so separating makes more sense.

**Comment by msarilar** [comment]
all right very well then. I was also thinking of people sharing the computer with fellow commanders :)

before I plan any release for that, do you confirm that **all** logs from **all** Elite Dangerous instances are located in the exact same folder? (I know it was implied in the original posts but I'd like to be very sure as I have no way to test it)

**Comment by msarilar** [comment]
well I published it in 1.0.0.28

don't hesitate to open new issues if you find problems with the handling of multiple commanders

**Comment by neotron** [comment]
I will check it out and report what I see! Thanks.
